OpenRA.Mods.Common/UtilityCommands/ExtractSettingsDocsCommand.cs

```diff
@@ -40,8 +40,14 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			Console.WriteLine("All settings can be changed by starting the game via a command-line parameter like `Game.Mod=ra`.");
 			Console.WriteLine();
 			Console.WriteLine("## Location");
-			Console.WriteLine("* Windows: `My Documents\\OpenRA\\settings.yaml`");
+			Console.WriteLine("* Windows: `%APPDATA%\\Roaming\\OpenRA\\settings.yaml`");
```

Comment:
Can we please use `\Users\<Username>\AppData\Roaming\OpenRA\` instead? I'm fairly sure that the average windows user won't know what %APPDATA% means.

Reply:
The problem with that was that it isn't a valid path. I can prepend a `C:` but even then `<Username>` will have to be replaced. I suppose most people just copy and paste the path?

Reply:
Copy and paste it where?

I took the path from the discord Orabot FWIW, so we go with something different we should update it there too.

Reply:
Copy and past the path to the explorer. At least that's what I'd do instead of typing.

Reply:
Using `%APPDATA%\Roaming\OpenRA` is known to work and you don't have to understand, just paste it into explorer. Also compare https://www.pcgamingwiki.com/wiki/OpenRA#Game_data

Reply:
Actually this is wrong:
```suggestion
			Console.WriteLine("* Windows: `%APPDATA%\\\OpenRA\\settings.yaml`");
```

On Windows Vista/7/8/10, `%APPDATA%` points to `C:\Users\(Username)\AppData\Roaming\`

Reply:
Oh derp, you're right.

Reply:
Updated and adjusted the wiki pages.